src/AdminModule/Components/Navbar/Navbar.tsx

```diff
@@ -1,7 +1,125 @@
-import React from 'react'
+import MenuIcon from "@mui/icons-material/Menu";
+import Container from "@mui/material/Container";
+import IconButton from "@mui/material/IconButton";
+import Tooltip from "@mui/material/Tooltip";
+import Button from '@mui/material/Button';
+import React, { useEffect, useState } from "react";
+import { baseUrl } from '../../../Constants/Components/Urls';
+import axios from "axios";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { Avatar } from "@mui/material";
+import defaultImage from '../../../assets/Avatar.png'
+
+
+
 
 export default function Navbar() {
+  const[userData,setUserData]=useState();
+//let loginData=localStorage.getItem('loginData');
+const loginData=JSON.parse(localStorage.getItem('loginData'))
+//console.log(loginData._id);
+let token = localStorage.getItem("adminToken");
+async function getUserDetails() {
+  try {
+    const response= await axios.get(`${baseUrl}/portal/users/${loginData._id}`, {
+      headers:
+         { Authorization: token },
+    });
+
+    setUserData(response.data.data.user);
+    console.log(response.data.data.user);
+    
+  } catch (error) {
+    console.error(error);
+  }
+  
+}
+useEffect(() => {
+  getUserDetails();
+}, []);
+
+
+  
+
   return (
-    <div>Navbar</div>
+   <>
+
+<Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static">
+        <Toolbar sx={{background:'#FFFFFF'}}>
+        <Typography
+           sx={{flexGrow:1}}
+            variant="h4"
+            component="div"
+            color="initial"
+          >
+            <Typography
+              variant=""
+             
+              style={{ fontSize: "clamp(1rem, 4vw, 2rem)",color:'#3d5afe', marginLeft:'70px' }}
+            >
+              Stay
+            </Typography>
+            cation.
+          </Typography>
+         <Avatar   src={
```

Comment:
we need to use `alt` attribute for the fallback of the image it could be something like `${userName}'s Avatar`.
This will be helpful for the fallback of the image. If there is a problem loading the image, the first letter of the alt will be displayed;
https://mui.com/material-ui/react-avatar/#fallbacks
![image](https://github.com/SalmaAboArab/Hotel-Management-System/assets/64327685/4873bd11-825f-47dd-bcf1-493514781604)